pythonforandroid/bootstrap.py

```diff
@@ -34,6 +35,33 @@ def copy_files(src_root, dest_root, override=True):
                 os.makedirs(dest_file)
 
 
+default_recipe_priorities = [
+    "webview", "sdl2", "service_only"  # last is highest
+]
+# ^^ NOTE: these are just the default priorities if no special rules
+# apply (which you can find in the code below), so basically if no
+# known graphical lib or web lib is used - in which case service_only
+# is the most reasonable guess.
+
+
+def _cmp_bootstraps_by_priority(a, b):
+    global default_recipe_priorities
```

Comment:
This is confusing me, why do you need the `global` keyword if you're not updating the reference of the object?

Reply:
Ohhh then I need to remove it!! I somehow thought it would be less confusing to make sure people realize it's not a local var, but I guess this is more confusing isn't it? I'll remove it :grinning: